### ISSUE TITLE
Determine how to add the EFSL to the Tagdocs that are generated

### ISSUE THREAD

**Issue by pnicolucci** [eclipse-ee4j/jstl-api#134]
As part of the Jakarta EE9 review ballot process it was noticed that the tagsdocs did not include the EFSL. We need to figure out a way to add this in an upcoming release of the tagdocs.

This was brought up here : https://www.eclipse.org/lists/jakarta.ee-spec/msg01234.html and I said I'd open an issue here: https://www.eclipse.org/lists/jakarta.ee-spec/msg01242.html

Looking at the Faces docs here they don't seem to include it either:  https://jakarta.ee/specifications/faces/3.0/vdldoc/

We're using tlddoc and the only thing added to the documentation is the following in each HTML page:

>         <small><i>
>             Output Generated by 
>             <a href="http://taglibrarydoc.dev.java.net/" target="_blank">Tag Library Documentation Generator</a>.
>             Java, JSP, and JavaServer Pages are trademarks or 
>             registered trademarks of Sun Microsystems, Inc. in the US and other
>             countries.  Copyright 2002-4 Sun Microsystems, Inc.
>             4150 Network Circle
>             Santa Clara, CA 95054, U.S.A.
>             All Rights Reserved. 
>             </i></small>

When Faces uses vdldoc they remove the generated by using a flag in that component : https://github.com/eclipse-ee4j/faces-api/blob/master/api/pom.xml#L567 the generated by for vdldoc differs from the one above and there is noway to turn that off in tlddoc.

So in summary I don't think we need to remove the generated by no one seemed to have an issue with that. We can look to see if there is any way to insert the EFSL.
